### PR TITLE
Bind grn_obj_is_key_accessor()

### PR DESCRIPTION
**Pull request by myokoym**
See #110 

@kou Could you review this pull request?

### CODE REVIEW

File: ext/groonga/rb-grn-object.c [hunk 1]
```diff
@@ -1682,6 +1682,32 @@ rb_grn_object_accessor_p (VALUE self)
...
+ *     `false` otherwise.
+ *
+ * @since 5.1.1
+ */
+static VALUE
+rb_grn_object_key_accessor_p (VALUE self)
+{
+    grn_ctx *context;
+    grn_obj *object;
+    grn_bool accessor_p = GRN_FALSE;
```
Comment (kou) [inline ext/groonga/rb-grn-object.c]: It should be `key_accessor_p`.

File: test/test-accessor.rb [hunk 2]
```diff
@@ -21,6 +21,7 @@ def setup
     setup_database
     @posts = Groonga::Hash.create(:name => "Posts", :key_type => "ShortText")
     @id = @posts.column("_id")
+    @key = @posts.column("_key")
```
Comment (kou) [inline test/test-accessor.rb]: We should reduce scope. `@key` is used only `#key_accessor?` test case. It should be defined there not here.

File: ext/groonga/rb-grn-object.c (hunk 1 above)
Comment (myokoym) [inline ext/groonga/rb-grn-object.c]: Thanks. I've fixed it.

File: ext/groonga/rb-grn-object.c [hunk 3]
```diff
@@ -1682,6 +1682,32 @@ rb_grn_object_accessor_p (VALUE self)
     return CBOOL2RVAL(accessor_p);
 }
 
+/*
+ * Checks whether the object is key accessor or not.
+ *
+ * @overload key_accessor?
+ *   @return [Boolean] `true` if the object is key accessor,
+ *     `false` otherwise.
+ *
```
Comment (kou) [inline ext/groonga/rb-grn-object.c]: Add example like the following:

```text
@example `true` case: `column("_key")` is an accessor and it's an accessor for `_key`
  Groonga::Schema.create_table("Users", :key_type => :short_text)
  users = Groonga["Users"]
  users.column("_key").key_accessor? # => true

@example `false` case: `column("_id")` is an accessor but it's not an accessor for `_key`
  Groonga::Schema.create_table("Users", :key_type => :short_text)
  users = Groonga["Users"]
  users.column("_id").key_accessor? # => false
```

File: test/test-accessor.rb (hunk 2 above)
Comment (myokoym) [inline test/test-accessor.rb]: Understood. I've fixed it.

File: ext/groonga/rb-grn-object.c (hunk 3 above)
Comment (myokoym) [inline ext/groonga/rb-grn-object.c]: I agree with that. I'll add examples.

Comment (myokoym) [inline ext/groonga/rb-grn-object.c]: I've added examples.

**Comment by myokoym** [comment]
I'm going to squash the commits and merge the pull request.

**Comment by myokoym** [comment]
Merged. Next, I'll send a pull request for `grn_obj_is_true()`.

**Pull request merged**